spec/factories/users.rb

```diff
@@ -22,7 +22,7 @@
       verified_at { nil }
       sec_id { '123498767' }
       participant_id { Faker::Number.number(digits: 8) }
-      birls_id { Faker::Number.number(digits: 10) }
+      birls_id { Faker::Number.number(digits: 9) }
       icn { '123498767V234859' }
```

Comment:
This is actually supposed to be 8/9 chars. Lighthouse errors if this is not 8/9 chars, so had to change this for lighthouse testing to work.